### ISSUE TITLE
Fix self intersection during stand up back

### ISSUE THREAD

**Issue by Narcha** [HULKs/hulk#945]
If one of our arms is stuck in front of the robot's body during stand up back, it fails to get up correctly.

For an example, see https://www.youtube.com/watch?v=mCxaV0YZQIs
In our game vs. DNT, first half, 2:05 remaining.
Watch the Nao closest to the center circle.

Idea: Use forward kinematics to calculate the position of the hands in robot coordinates. Consider the x component (pointing forwards) and the y component (pointing left) to test if the hand is in front of the robot, and not to its side. If it is stuck, move it out to the side before starting the stand up motion. Maybe a similar issue also exists for stand up front or other motions.

**Comment by rmburg** [comment]
no longer relevant with the K1
